src/main/features/ccj/routes/pay-by-set-date.ts

```diff
@@ -4,14 +4,15 @@ import { Paths } from 'ccj/paths'
 import { Form } from 'forms/form'
 import { FormValidator } from 'forms/validation/formValidator'
 import { PayBySetDate } from 'ccj/form/models/payBySetDate'
-import User from 'idam/user'
+import { User } from 'idam/user'
 import { DraftService } from 'services/draftService'
 import { ErrorHandling } from 'common/errorHandling'
 
 function renderView (form: Form<PayBySetDate>, res: express.Response): void {
   res.render(Paths.payBySetDatePage.associatedView, { form: form })
 }
 
+/* tslint:disable:no-default-export */
```

Comment:
any plan on a way to address this?

Reply:
Not at this moment. I would need to think how to rewrite routes so that we can have a nice code without default exports.